### ISSUE TITLE
Modify definition of major and minor change/version

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#745]
Currently we say that a major change is any modification that impacts inferencing. We have discussed loosening this to consider as minor changes those in which inferencing is impacted but in a way that the overwhelming majority of users will not care about. 

Example: in gist 11.1.0 we removed the restriction on `PhysicallyIdentifiableItem` stating that it must have an identifier (i.e., it's a subclass of the class of all things have an identifier). Technically, this impacts inferencing: if anyone was counting on being able to infer that a PID has an identifier (regardless of whether the identifier is known or not), they would be adversely affected by this change. However, we deem it highly unlikely that anyone _does_ count on this, and therefore we can consider it a minor change.

This introduces subjectivity into the major/minor distinction, but we believe the benefit of not having to postpone such changes to a major release is greater than the cost.

**Comment by dylan-sa** [comment]
I like the distinction between major and minor impacts to inference, especially if it allows us to include more changes in a minor release. If we continue to use this as a guide to the major/minor release division, would it be helpful for us to develop a list of the types of changes to inference that generally rise to the level of a major impact? I know we say a little about this in the [change and release management documentation](https://github.com/semanticarts/gist/blob/develop/docs/ChangeAndReleaseManagement.md) already, but more examples would be helpful for quicker categorization of changes in the future.

**Comment by uscholdm** [comment]
I agree to having a category for minor impact of inference change.

**Comment by rjyounes** [comment]
@dylan-sa 

>would it be helpful for us to develop a list of the types of changes to inference that generally rise to the level of a major impact? I know we say a little about this in the [change and release management documentation](https://github.com/semanticarts/gist/blob/develop/docs/ChangeAndReleaseManagement.md) already, but more examples would be helpful for quicker categorization of changes in the future.

Yes, that's a good idea. In some cases it's going to go down to subjective judgment, though.

**Comment by uscholdm** [comment]
> It wouldn't technically be a category, because such changes would be considered minor; it's just a redefinition of what constitutes a major or minor change.

Yes. ok.

**Comment by rjyounes** [comment]
Closing - already done.
